zqd/space/space.go

```diff
@@ -48,7 +50,7 @@ func Open(root, name string) (*Space, error) {
 		}
 		return nil, err
 	}
-	return &Space{path, c}, nil
+	return &Space{path, c, bzngio.NewTimeIndex()}, nil
```

Comment:
It looks like we call space.Open for every search right now. Doesn't that mean the index will be recreated on every search?

Reply:
Ugh.  How about if I leave out the integration with zqd for now since fixing this entails bigger changes to space.go

Reply:
Sounds good to me.